lib/main.dart

```diff
@@ -11,24 +13,38 @@ import 'package:screenshot/screenshot.dart';
 import 'app/services/navigation.dart';
 import 'app/widgets/hpi_theme.dart';
 
-Future<ByteData> fetchFont(String url) async {
-  try { 
-    final response = await http.get(url);
-      if (response.statusCode == 200)
-    return ByteData.view(response.bodyBytes.buffer);
-  else
-    throw Exception('Failed to load font');
-  } on SocketException catch (_) {
-      var fontLoader = FontLoader('HpiIcons').load();
+const fontUrl =
+    'https://hpi.de/fileadmin/templates/fonts/9de9709d-f77a-44ad-96b9-6fea586f7efb.ttf';
+
+Future<ByteData> _downloadFontToCache() async {
+  final file = File('${(await getTemporaryDirectory()).path}/NeoSans.ttf');
+
+  if (await file.exists()) {
+    // We already downloaded a cached version of the font, so just use that.
+    final bytes = file.readAsBytesSync();
+    return ByteData.view(bytes.buffer);
+  } else {
+    // Download the font.
+    final response = await http.get(fontUrl);
+    file.writeAsBytes(response.bodyBytes);
+    if (response.statusCode == 200)
+      return ByteData.view(response.bodyBytes.buffer);
+    else
```

Comment:
This branch should be removed for the moment, as loading the font should fail silently. Once we have custom error handling, it could be added back in.